### ISSUE TITLE
[BUG] Can't install chrome using npx playwright install chrome

### ISSUE THREAD

**Issue by cardinalX** [microsoft/playwright#11300]
**Context:** I'm using command npx playwright install chrome in docker. But today, after a long holidays, I found that the command began to fail with an error.
- Playwright Version: 1.17.1/1.16.3
- Operating System: Mac OS intel
- Node.js version: 14.17.6
- Browser: Chrome
- Extra: used Docker images `mcr.microsoft.com/playwright:v1.16.3-focal` and `mcr.microsoft.com/playwright:v1.17.1-focal`

I use package.json with v.1.17.2 and v.1.16.3 (result is identical)
```
  "devDependencies": {
    "@playwright/test": "1.17.2",
    "allure-playwright": "^2.0.0-beta.14"
  },
```

For example dockerfile `FROM mcr.microsoft.com/playwright:v1.16.3-focal` fails when `RUN google-chrome --version`

```
FROM mcr.microsoft.com/playwright:v1.16.3-focal as Tester
WORKDIR /tester
COPY tests/package.json ./

RUN npm install
RUN npx playwright install chrome
```

### Error result:
```
#9 8.050  87400K .......... .......... .......... .......... .......... 99% 61.0M 0s
#9 8.051  87450K .......... .......... .......... .......... .......... 99% 60.0M 0s
#9 8.052  87500K .......... .......... .......... .......... .......... 99% 76.2M 0s
#9 8.053  87550K .......... .......... .......... .......... .......... 99% 70.4M 0s
#9 8.053  87600K .......... .......... .......... .......... .......... 99% 42.9M 0s
#9 8.054  87650K .......... .......... .......... .......... .......... 99% 49.8M 0s
#9 8.055  87700K .......... .......... .......... .......... ........  100% 80.4M=4.6s
#9 8.056 
#9 8.056 2022-01-10 16:56:34 (18.5 MB/s) - 'google-chrome-stable_current_amd64.deb' saved [89854380/89854380]
#9 8.056 
#9 8.057 + apt-get install -y ./google-chrome-stable_current_amd64.deb
#9 8.089 Reading package lists...
#9 8.890 Building dependency tree...
#9 9.055 Reading state information...
#9 9.221 The following additional packages will be installed:
#9 9.221   libauthen-sasl-perl libdata-dump-perl libencode-locale-perl
#9 9.221   libfile-basedir-perl libfile-desktopentry-perl libfile-listing-perl
#9 9.221   libfile-mimeinfo-perl libfont-afm-perl libhtml-form-perl libhtml-format-perl
#9 9.221   libhtml-parser-perl libhtml-tagset-perl libhtml-tree-perl
#9 9.221   libhttp-cookies-perl libhttp-daemon-perl libhttp-date-perl
#9 9.221   libhttp-message-perl libhttp-negotiate-perl libio-html-perl
#9 9.222   libio-socket-ssl-perl libio-stringy-perl libipc-system-simple-perl
#9 9.222   liblwp-mediatypes-perl liblwp-protocol-https-perl libmailtools-perl
#9 9.222   libnet-dbus-perl libnet-http-perl libnet-smtp-ssl-perl libnet-ssleay-perl
#9 9.222   libtie-ixhash-perl libtimedate-perl libtry-tiny-perl libu2f-udev liburi-perl
#9 9.222   libwww-perl libwww-robotrules-perl libx11-protocol-perl libxml-parser-perl
#9 9.222   libxml-twig-perl libxml-xpathengine-perl netbase perl-openssl-defaults udev
#9 9.222   x11-xserver-utils xdg-utils
#9 9.224 Suggested packages:
#9 9.224   libdigest-hmac-perl libgssapi-perl libcrypt-ssleay-perl libauthen-ntlm-perl
#9 9.224   libunicode-map8-perl libunicode-string-perl xml-twig-tools nickle cairo-5c
#9 9.224   xorg-docs-core
#9 9.300 The following NEW packages will be installed:
#9 9.300   google-chrome-stable libauthen-sasl-perl libdata-dump-perl
#9 9.300   libencode-locale-perl libfile-basedir-perl libfile-desktopentry-perl
#9 9.300   libfile-listing-perl libfile-mimeinfo-perl libfont-afm-perl
#9 9.300   libhtml-form-perl libhtml-format-perl libhtml-parser-perl
#9 9.300   libhtml-tagset-perl libhtml-tree-perl libhttp-cookies-perl
#9 9.300   libhttp-daemon-perl libhttp-date-perl libhttp-message-perl
#9 9.301   libhttp-negotiate-perl libio-html-perl libio-socket-ssl-perl
#9 9.301   libio-stringy-perl libipc-system-simple-perl liblwp-mediatypes-perl
#9 9.301   liblwp-protocol-https-perl libmailtools-perl libnet-dbus-perl
#9 9.304   libnet-http-perl libnet-smtp-ssl-perl libnet-ssleay-perl libtie-ixhash-perl
#9 9.304   libtimedate-perl libtry-tiny-perl libu2f-udev liburi-perl libwww-perl
#9 9.307   libwww-robotrules-perl libx11-protocol-perl libxml-parser-perl
#9 9.312   libxml-twig-perl libxml-xpathengine-perl netbase perl-openssl-defaults udev
#9 9.312   x11-xserver-utils xdg-utils
#9 9.463 0 upgraded, 46 newly installed, 0 to remove and 0 not upgraded.
#9 9.463 Need to get 4002 kB/93.9 MB of archives.
#9 9.463 After this operation, 303 MB of additional disk space will be used.
#9 9.463 Get:1 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 xdg-utils all 1.1.3-2ubuntu1.20.04.2 [61.4 kB]
#9 9.653 Get:2 http://archive.ubuntu.com/ubuntu focal/main amd64 netbase all 6.1 [13.1 kB]
#9 9.657 Err:3 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 udev amd64 245.4-4ubuntu3.13
#9 9.657   404  Not Found [IP: 91.189.88.152 80]
#9 9.658 Get:4 http://archive.ubuntu.com/ubuntu focal/main amd64 libdata-dump-perl all 1.23-1 [27.0 kB]
#9 9.672 Get:5 http://archive.ubuntu.com/ubuntu focal/main amd64 libencode-locale-perl all 1.05-1 [12.3 kB]
#9 9.674 Get:6 http://archive.ubuntu.com/ubuntu focal/main amd64 libipc-system-simple-perl all 1.26-1 [22.8 kB]
#9 9.710 Get:7 http://archive.ubuntu.com/ubuntu focal/main amd64 libfile-basedir-perl all 0.08-1 [16.9 kB]
#9 9.721 Get:8 http://archive.ubuntu.com/ubuntu focal/main amd64 liburi-perl all 1.76-2 [77.5 kB]
#9 9.747 Get:9 http://archive.ubuntu.com/ubuntu focal/main amd64 libfile-desktopentry-perl all 0.22-1 [18.2 kB]
#9 9.751 Get:10 http://archive.ubuntu.com/ubuntu focal/main amd64 libtimedate-perl all 2.3200-1 [34.0 kB]
#9 9.773 Get:11 http://archive.ubuntu.com/ubuntu focal/main amd64 libhttp-date-perl all 6.05-1 [9920 B]
#9 9.776 Get:12 http://archive.ubuntu.com/ubuntu focal/main amd64 libfile-listing-perl all 6.04-1 [9774 B]
#9 9.779 Get:13 http://archive.ubuntu.com/ubuntu focal/main amd64 libfile-mimeinfo-perl all 0.29-1 [41.5 kB]
#9 9.789 Get:14 http://archive.ubuntu.com/ubuntu focal/main amd64 libfont-afm-perl all 1.20-2 [13.2 kB]
#9 9.791 Get:15 http://archive.ubuntu.com/ubuntu focal/main amd64 libhtml-tagset-perl all 3.20-4 [12.5 kB]
#9 9.795 Get:16 http://archive.ubuntu.com/ubuntu focal/main amd64 libhtml-parser-perl amd64 3.72-5 [86.3 kB]
#9 9.812 Get:17 http://archive.ubuntu.com/ubuntu focal/main amd64 libio-html-perl all 1.001-1 [14.9 kB]
#9 9.815 Get:18 http://archive.ubuntu.com/ubuntu focal/main amd64 liblwp-mediatypes-perl all 6.04-1 [19.5 kB]
#9 9.820 Get:19 http://archive.ubuntu.com/ubuntu focal/main amd64 libhttp-message-perl all 6.22-1 [76.1 kB]
#9 9.832 Get:20 http://archive.ubuntu.com/ubuntu focal/main amd64 libhtml-form-perl all 6.07-1 [22.2 kB]
#9 9.836 Get:21 http://archive.ubuntu.com/ubuntu focal/main amd64 libhtml-tree-perl all 5.07-2 [200 kB]
#9 9.860 Get:22 http://archive.ubuntu.com/ubuntu focal/main amd64 libhtml-format-perl all 2.12-1 [41.3 kB]
#9 9.864 Get:23 http://archive.ubuntu.com/ubuntu focal/main amd64 libhttp-cookies-perl all 6.08-1 [18.3 kB]
#9 9.865 Get:24 http://archive.ubuntu.com/ubuntu focal/main amd64 libhttp-daemon-perl all 6.06-1 [21.4 kB]
#9 9.867 Get:25 http://archive.ubuntu.com/ubuntu focal/main amd64 libhttp-negotiate-perl all 6.01-1 [12.5 kB]
#9 9.870 Get:26 http://archive.ubuntu.com/ubuntu focal/main amd64 perl-openssl-defaults amd64 4 [7192 B]
#9 9.874 Get:27 http://archive.ubuntu.com/ubuntu focal/main amd64 libnet-ssleay-perl amd64 1.88-2ubuntu1 [291 kB]
#9 9.898 Get:28 http://archive.ubuntu.com/ubuntu focal/main amd64 libio-socket-ssl-perl all 2.067-1 [176 kB]
#9 9.908 Get:29 http://archive.ubuntu.com/ubuntu focal/main amd64 libio-stringy-perl all 2.111-3 [55.8 kB]
#9 9.911 Get:30 http://archive.ubuntu.com/ubuntu focal/main amd64 libnet-http-perl all 6.19-1 [22.8 kB]
#9 9.919 Get:31 http://archive.ubuntu.com/ubuntu focal/main amd64 libtry-tiny-perl all 0.30-1 [20.5 kB]
#9 9.923 Get:32 http://archive.ubuntu.com/ubuntu focal/main amd64 libwww-robotrules-perl all 6.02-1 [12.6 kB]
#9 9.925 Get:33 http://archive.ubuntu.com/ubuntu focal/main amd64 libwww-perl all 6.43-1 [140 kB]
#9 9.931 Get:34 http://archive.ubuntu.com/ubuntu focal/main amd64 liblwp-protocol-https-perl all 6.07-2ubuntu2 [8560 B]
#9 9.933 Get:35 http://archive.ubuntu.com/ubuntu focal/main amd64 libnet-smtp-ssl-perl all 1.04-1 [5948 B]
#9 9.933 Get:36 http://archive.ubuntu.com/ubuntu focal/main amd64 libmailtools-perl all 2.21-1 [80.7 kB]
#9 9.943 Get:37 /tmp/google-chrome-stable_current_amd64.deb google-chrome-stable amd64 97.0.4692.71-1 [89.9 MB]
#9 9.959 Get:38 http://archive.ubuntu.com/ubuntu focal/main amd64 libxml-parser-perl amd64 2.46-1 [193 kB]
#9 9.967 Get:39 http://archive.ubuntu.com/ubuntu focal/main amd64 libxml-twig-perl all 1:3.50-2 [155 kB]
#9 9.974 Get:40 http://archive.ubuntu.com/ubuntu focal/main amd64 libnet-dbus-perl amd64 1.2.0-1 [177 kB]
#9 9.981 Get:41 http://archive.ubuntu.com/ubuntu focal/main amd64 libtie-ixhash-perl all 1.23-2 [11.2 kB]
#9 9.982 Get:42 http://archive.ubuntu.com/ubuntu focal/main amd64 libu2f-udev all 1.1.10-1 [6108 B]
#9 9.985 Get:43 http://archive.ubuntu.com/ubuntu focal/main amd64 libx11-protocol-perl all 0.56-7 [149 kB]
#9 9.991 Get:44 http://archive.ubuntu.com/ubuntu focal/main amd64 libxml-xpathengine-perl all 0.14-1 [31.8 kB]
#9 9.993 Get:45 http://archive.ubuntu.com/ubuntu focal/main amd64 x11-xserver-utils amd64 7.7+8 [162 kB]
#9 9.999 Get:46 http://archive.ubuntu.com/ubuntu focal/main amd64 libauthen-sasl-perl all 2.1600-1 [48.7 kB]
#9 10.01 E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/s/systemd/udev_245.4-4ubuntu3.13_amd64.deb  404  Not Found [IP: 91.189.88.152 80]
#9 10.01 E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
#9 10.01 Fetched 2637 kB in 1s (3908 kB/s)
#9 10.01 Failed to install browsers
#9 10.01 Error: Failed to install chrome
------
executor failed running [/bin/sh -c npx playwright install chrome]: exit code: 1
```

Using `FROM mcr.microsoft.com/playwright:v1.17.1-focal` fails when `RUN google-chrome --version`
with another error

```
FROM mcr.microsoft.com/playwright:v1.17.1-focal as Tester
WORKDIR /tester
COPY tests/package.json ./

RUN npm install
RUN npx playwright install chrome
```

Error result:
```
#9 6.797  87650K .......... .......... .......... .......... .......... 99% 12.7M 0s
#9 6.801  87700K .......... .......... .......... .......... ........  100% 18.5M=5.4s
#9 6.803 
#9 6.803 2022-01-10 17:13:07 (15.8 MB/s) - 'google-chrome-stable_current_amd64.deb' saved [89854380/89854380]
#9 6.803 
#9 6.805 + apt-get install -y ./google-chrome-stable_current_amd64.deb
#9 6.811 Reading package lists...
#9 6.825 Building dependency tree...
#9 6.826 Reading state information...
#9 6.828 Some packages could not be installed. This may mean that you have
#9 6.828 requested an impossible situation or if you are using the unstable
#9 6.828 distribution that some required packages have not yet been created
#9 6.828 or been moved out of Incoming.
#9 6.828 The following information may help to resolve the situation:
#9 6.828 
#9 6.828 The following packages have unmet dependencies:
#9 6.829  google-chrome-stable : Depends: xdg-utils (>= 1.0.2) but it is not installable
#9 6.829                         Recommends: libu2f-udev but it is not installable
#9 6.830 E: Unable to correct problems, you have held broken packages.
#9 6.835 Failed to install browsers
#9 6.835 Error: Failed to install chrome
------
executor failed running [/bin/sh -c npx playwright install chrome]: exit code: 1
```

**Comment by mxschmitt** [comment]
We fixed this one already, you can either use our latest dev release `npm install @playwright/test@next` or add the workaround: 'add before npx playwright install chrome "apt-get update"'

**Comment by dgozman** [comment]
Closing since the problem should be fixed for v1.18.
